biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.css

```diff
@@ -35,6 +35,30 @@ label {
   text-align: left;
 }
 
+.navbar {
```

Comment:
navbars generally cover the entire page top 
what about importing an existing one? https://react-bootstrap.github.io/components/navbar/